server/api/search.ts

```diff
@@ -22,7 +22,7 @@ export default defineEventHandler(async (event) => {
         // Only use `.md` files which are not drafts and has content
           return doc?._extension === 'md' &&
           doc?._draft === false &&
-          doc?._empty === false
+          (doc?._empty === undefined || doc?._empty === false)
```

Comment:
Could we simplify it to this?

```suggestion
          !doc?._empty
```

Reply:
Sure. Done!